### ISSUE TITLE
Document a standard crash reporting integration

### ISSUE THREAD

**Issue by xster** [flutter/flutter#12809]
Probably after #9592 and #4070 are done.

**Comment by yjbanov** [comment]
I've never written one.

**Comment by neiljaywarner** [comment]
@Hixie @eseidelGoogle is there an update on this? one of our leads was asking about this, and I know there's a lot of different places discussing it...

**Comment by eseidelGoogle** [comment]
I'm not aware of progress on docs.  However, we are looking into https://github.com/flutter/flutter/issues/14765 at the moment and that work may result in some.

FYI @sfshaza2 this may belong in the flutter/website tracker.

**Comment by neiljaywarner** [comment]
@eseidelGoogle @sfshaza2 this might be especially helpful if there are quirks with a roadmap, and as add2app and enterprise are moving along.

I'm confused as to whether to use official flutter team crashlytics plugin or previously existing mature plugin, maybe there will be a 'cutover' date and the non-official one can be deprecated when the official one is stable, or the official one can mention when there is feature parity and stability, or the differences, etc.

Of particular note:
* Do keys work as expected?
* "Dart error" is confusing and/or suboptimal, i filed another issue for that.

other than that it's awesome i'm happy to see that it is there, and i'm *very* happy it seemed to work with add2app with zero changes between Fabric crashlytics and firebase crashlytics (although I haven't seen keys work with fabric.io...)

**Comment by eseidelGoogle** [comment]
FYI @kroikie who wrote firebase_crashlytics.

**Comment by neiljaywarner** [comment]
I'm also wondering if it can be modified so that in the likely event add2app already has crashlytics that the plugin uses that version instead....
I know it's a new plugin
Thanks for doing it
.

**Comment by kroikie** [comment]
@xster

This issue has been moved to https://github.com/FirebaseExtended/flutterfire/issues/949. Any further collaboration will be done there.
